applications/StructuralMechanicsApplication/custom_elements/membrane_element.cpp

```diff
@@ -1154,13 +1153,7 @@ void MembraneElement::AddExplicitContribution(
             array_1d<double, 3>& r_nodal_inertia = GetGeometry()[i].GetValue(NODAL_INERTIA);
```

Comment:
@KlausBSautter maybe you can take another look, now `r_nodal_inertia` is unused, you sure it is correct?